pkgs/development/python-modules/pytorch-ocl/default.nix

```diff
@@ -0,0 +1,57 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  cmake,
+  torch,
+  ocl-icd,
+  opencl-clhpp,
+  sqlite,
+  python,
+}:
+buildPythonPackage rec {
```

Comment:
```suggestion
buildPythonPackage (finalAttrs: {
```

Reply:
Ok, will try that. I only saw `buildPythonPackage` called on attrsets so far.

Reply:
With this one I am not sure, I am still fairly new to nixpkgs, but here it was the same with the best practices that rec can lead to some infinite recursions, while the other can't. Not sure if there are some special cases for certain packages. 

Maybe you don't want to override the attributes of this one so it doesn't matter? 

It was just a suggestion I've seen other people make, and that was suggested in my recent prs

Reply:
Looks like it doesn't work with `buildPythonPackage`: https://gist.github.com/GrahamcOfBorg/c420f0af916990df0933712df3fb2775 (`error: value is a function while a set was expected`). Will revert that change now.

Hope it still can be nicely overridden when someone needs it. Didn't have any issues overriding packages built with `buildPythonPackage rec...` so far, though.

Reply:
Thanks for trying it, I've learned something new as well now